templates/vars.yml.j2

```diff
@@ -181,3 +181,6 @@ matrix_nginx_proxy_proxy_synapse_additional_server_configuration_blocks:
   auth_basic "protected";
   auth_basic_user_file /nginx-data/matrix-synapse-metrics-htpasswd;
   }'
+
+
+matrix_synapse_container_extra_arguments: ['-m=2g']
```

Comment:
I think this extra argument should be appended to a possible list, otherwise, it will override other configurations like decentraland pass auth provider